workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-article/ddm-template.ftl

```diff
@@ -159,6 +158,8 @@
 						</div>
 					</#list>
 				</div>
+
+				<div class="learn-article-recipe-recomendations" data-article-id="${.vars["reserved-article-id"].data}" id="feature-recipes-placeholder"></div>
```

Comment:
Where the class `learn-article-recipe-recomendations` is defined?
If necessary, rename to related-recipes.

Reply:
`id="feature-recipes-placeholder"`
please rename to `article-related-recipes`.